web/src/components/Inputs/Button.tsx

```diff
@@ -0,0 +1,50 @@
+import { AnimatePresence } from "framer-motion";
+import { ButtonHTMLAttributes, ReactNode } from "react";
+
+import CircularProgress from "../Feedback/CircularProgress";
+
+interface Props extends ButtonHTMLAttributes<HTMLButtonElement> {
+  disabled?: boolean;
+  loading?: boolean;
+  className?: string;
+  children: ReactNode;
+}
+
+export default function Button({
+  disabled = false,
+  loading = false,
+  className = "",
+  children,
+  ...rest
+}: Props) {
+  //#region Styles
```

Comment:
Some IDEs like Webstorm will recognize this and allow you to collapse/expand regions. It also adds readability so that you know where to put hooks, variables etc

Reply:
Ok fair enough. I'm not a huge fan of editor-specific comments but these seems fairly benign.